lib/perfdata/influxdbwriter.cpp

```diff
@@ -381,45 +358,60 @@ void InfluxdbWriter::SendMetric(const Dictionary::Ptr& tmpl, const String& label
 			else
 				msgbuf << ",";
 
-			msgbuf << EscapeKey(pair.first) << "=" << EscapeField(pair.second);
+			msgbuf << EscapeKeyOrTagValue(pair.first) << "=" << EscapeValue(pair.second);
 		}
 	}
 
 	msgbuf << " " <<  static_cast<unsigned long>(ts);
 
+#ifdef _DEBUG
 	Log(LogDebug, "InfluxdbWriter")
 	    << "Add to metric list: '" << msgbuf.str() << "'.";
+#endif /* _DEBUG */
 
-	// Atomically buffer the data point
-	boost::mutex::scoped_lock lock(m_DataBufferMutex);
-	m_DataBuffer.push_back(String(msgbuf.str()));
+	// Buffer the data point
+	m_DataBuffer.push_back(msgbuf.str());
 
 	// Flush if we've buffered too much to prevent excessive memory use
 	if (static_cast<int>(m_DataBuffer.size()) >= GetFlushThreshold()) {
 		Log(LogDebug, "InfluxdbWriter")
 		    << "Data buffer overflow writing " << m_DataBuffer.size() << " data points";
-		Flush();
+
+		double st = Utility::GetTime();
+
+		try {
+			Flush();
+		} catch (...) {
+			/* Do nothing. */
+		}
+
+		double et = Utility::GetTime();
+		Log(LogInformation, "InfluxDbWriter")
```

Comment:
That might be too noisy as information level.

Reply:
Oops, that's not even meant to be there at all, at least not in the PR. 😇